### DISCUSSION TITLE
Cannot install on MacOS 12.3.1

### DISCUSSION THREAD

**Discussion by jjriffle** [pawelsalawa/sqlitestudio#4451] [Q&A]
When clicking the download to install I receive this message on both my Apple Chip iMac and Intel Chip MacBook Pro. 

“SQLiteStudio” cannot be opened because the developer cannot be verified.
macOS cannot verify that this app is free from malware.
This item is on the disk image “sqlitestudio-3.3.3.dmg”. Safari downloaded this disk image today at 10:57 AM from objects.githubusercontent.com.

### Steps to reproduce
Download Mac version
Click download to install

### Operating system
macOS Monterey 12.3.1

### SQLiteStudio version
3.3.3

**Comment by pawelsalawa** [comment]
Right-click on the app, choose "Open". Then in the same warning dialog you will have an option (a button in bottom corner) to "Open" the app anyway.

**Reply by jjriffle** [reply]
Thank you SO much!

**Reply by tchaffee** [reply]
@pawelsalawa any plans to become a verified developer for those of us at companies with security policies that make the workaround disadvantageous?

**Reply by pawelsalawa** [reply]
#3932

**Reply by ghost** [reply]
is this still queued?

**Reply by pawelsalawa** [reply]
I would say it's postponed indefinitely, rather then queued. See #3932 for more details.

**Reply by Marek84l** [reply]
Thank you!!!! It worked. You saved me.

**Comment by ozmium** [comment]
The workaround of **right-click on the Installer -> Open**, does not work on MacOS 12 Monterey or MacOS 13 Ventura. There is no button or option to <kbd>"Open Anyway"</kbd>. The dialog only shows the options of <kbd>Eject Disk Image</kbd> and <kbd>Cancel</kbd>.

Furthermore, clicking on **System Preferences -> Security & Privacy -> Open Anyway** also does not work. It only shows the same options of <kbd>Eject Disk Image</kbd> and <kbd>Cancel</kbd>.

So it's not possible to install this app for most people.

![SQLite Studio 3 4 4, macOS 12 Monterey, 2023-10-20](https://github.com/pawelsalawa/sqlitestudio/assets/6191136/5599831c-0380-4201-9e70-21a4f9d5cb3d)

**Reply by pawelsalawa** [reply]
How about clicking the lock on the bottom, entering password and then clicking "Allow" (just below the Open Anyway)?

**Reply by Yanyu-Chen1010** [reply]
<img width="444" alt="Screenshot 2023-12-13 at 11 26 26" src="https://github.com/pawelsalawa/sqlitestudio/assets/70660174/60a50a90-a33c-4b39-a0de-0519a076bfca">
 
This instruction works for me, hope it helps.

**Reply by profcommie** [reply]
Clicking the Open Anyway button in your screenshot, found in the Settings>Security & Privacy menu caused the error window to relaunch with the option to click open.
This worked to allow me to install.

**Comment by jacoblukewood** [comment]
The only solution that worked for me is ⌘ Command + ⌥ Option + O. This seems to do something different to just the standard right click and open that should work but doesn't.
